app/src/views/Login.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import {View, Text, StyleSheet, TextInput, Button} from 'react-native'
+
+import { Main } from './navigation'
+
+export default class Login extends React.component {
```

Comment:
I'm not sure what this should look like integrated with the Authentication. I'd get with Titus and see whats up

Reply:
gotcha